lib/constants.js

```diff
@@ -17,3 +17,5 @@ export const HEADERS = {
   remaining: 'x-slow-down-remaining',
   delay: 'x-slow-down-delay'
 }
+
+export const REDIS_KEY_TTL_IS_NOT_SET = -1
```

Comment:
as this is something related only to redis, IMO it should be declared directly inside the `redisStore` file